buildenv/jenkins/jobs/infrastructure/copyrightCheckDir.sh

```diff
@@ -0,0 +1,612 @@
+      else
+        if [ "$FOUND_ORACLE_DESIGNATES" -gt 0 ]; then
+          FOUND_ORACLE_CPE=$FOUND_CPE
+          trace "We have found the Oracle CPE on line $FOUND_ORACLE_CPE in file: $1"
+        fi
+      fi
+    fi
+
+    FOUND_GPLV2=$(head -n $MAX_LINES "$1" | $GREP -n "GNU General Public License" | head -n 1 | cut -d: -f1)
+    [ -z "$FOUND_GPLV2" ] && FOUND_GPLV2=0
+    if [ "$FOUND_GPLV2" -gt 0 ]; then
+      trace "We have found GPLv2 on line $FOUND_GPLV2 in file: $1"
+    fi
+
+    FOUND_NON_IBM_COPYRIGHT=0
+    [ "$FOUND_APACHE_COPYRIGHT" -gt 0 ]     && FOUND_NON_IBM_COPYRIGHT=1
+    [ "$FOUND_BSD_OR_MIT_COPYRIGHT" -gt 0 ] && FOUND_NON_IBM_COPYRIGHT=1
+    [ "$FOUND_ORACLE_COPYRIGHT" -gt 0 ]     && FOUND_NON_IBM_COPYRIGHT=1
+    [ "$FOUND_COPYRIGHT" -gt 0 ]            && FOUND_NON_IBM_COPYRIGHT=1
+    if [ "$FOUND_NON_IBM_COPYRIGHT" -eq 1 ]; then
+      trace "We have found a non IBM copyright"
+    fi
+
+    # Check to see whether the file is in the built JDK
+    IN_JDK=1
+    case "$1" in
+#      */solaris/*)
+#        echo  "$1 deemed not to be in the built JDK because it matches case parameter expansion */solaris/*"
+#        IN_JDK=0;;
+#      *make/*)
+#        echo  "$1 deemed not to be in the built JDK because it matches case parameter expansion *make/*"
+#        IN_JDK=0;;
+      test/*)
+        trace "$1 deemed not to be in the built JDK because it matches case parameter expansion test/*"
+        IN_JDK=0;;
+      bin/*)
+        trace "$1 deemed not to be in the built JDK because it matches case parameter expansion bin/*"
+        IN_JDK=0;;
+      make/autoconf/*)
+        trace  "$1 deemed not to be in the built JDK because it matches case parameter expansion make/autoconf/*"
+        IN_JDK=0;;
+      make/hotspot/*)
+        trace  "$1 deemed not to be in the built JDK because it matches case parameter expansion make/hotspot/*)"
+        IN_JDK=0;;
+      make/langtools/*)
+        trace  "$1 deemed not to be in the built JDK because it matches case parameter expansion make/langtools/*"
+        IN_JDK=0;;
+      make/nashorn/*)
+        trace  "$1 deemed not to be in the built JDK because it matches case parameter expansion make/nashorn/*"
+        IN_JDK=0;;
+      make/scripts/*)
+        trace  "$1 deemed not to be in the built JDK because it matches case parameter expansion make/scripts/*"
+        IN_JDK=0;;
+      make/templates/*)
+        trace  "$1 deemed not to be in the built JDK because it matches case parameter expansion make/templates/*"
+        IN_JDK=0;;
+      src/utils/*)
+        trace  "$1 deemed not to be in the built JDK because it matches case parameter expansion src/utils/*"
+        IN_JDK=0;;
+      */test/*)
+        trace  "$1 deemed not to be in the built JDK because it matches case parameter expansion */test/*"
+        IN_JDK=0;;
+#      */doc/*)
+#        trace  "$1 deemed not to be in the built JDK because it matches case parameter expansion */doc/*"
+#        IN_JDK=0;;
```

Comment:
Commented intentionally?

Reply:
Yes. Turned out that only the .1 files needed to be excluded, so a specific pattern was used for those. Removed the commented out lines.